### ISSUE TITLE
Address skipped heading level 

### ISSUE THREAD

**Issue by jenndiaz** [wwcodecolorado/scrollodex#105]
#### Description
The project currently skips from H1 to and H3. This does not meet WCAG and could causing a confusing navigation for screenreader users. 

#### How to visualize page structure with Wave 
1. install wave web accessibility evaluation tool browser extension 
2. run tool (on Mac `cmd + shift + u`) 
3. within summary sidebar, select structure tab 
4. Note the Orange alert concerning skipped heading level

#### Screenshot(s)
![screenshot](https://user-images.githubusercontent.com/63808889/195728615-85aa83e8-c379-4c88-ab97-247d46ca283e.jpg)

#### Tasks
- [ ] Address the skipped heading level 
- [ ] Ensure fix does not change styling

#### Resources 
[Deque Heading Order](https://dequeuniversity.com/rules/axe/4.4/heading-order)
